PrepareVicuna.md

```diff
@@ -24,11 +24,14 @@ First, Install their library that is compatible with v0 Vicuna by
 ```
 pip install git+https://github.com/lm-sys/FastChat.git@v0.1.10
```

Comment:
fyi: it requires [fschat>=0.2.1](https://github.com/lm-sys/FastChat/blob/main/docs/vicuna_weights_version.md)